packages/codebytes/src/libs/eventTracking.ts

```diff
@@ -0,0 +1,16 @@
+import { createTracker } from '@codecademy/tracking';
+
+const IS_DEV = process.env.NODE_ENV === 'development';
+const API_BASE_URL =
+  process.env.GATSBY_MONOLITH_API_BASE || 'https://www.codecademy.com';
```

Comment:
we should probably rename GATSBY_MONOLITH_API_BASE since we wont be using gatsby anymore, and we should add the var to the env file here

Reply:
I think we have two options here. We could either:

1. Set this to `https://www.codecademy.com` in this file. The only limitation here, is that all of our tracking events would run on segment production, even if we were QA'ing in a staging environment. 

2. Set the environment variable as something like `MONOLITH_API_BASE` in the host .env file, both in the monolith and in static sites. We'd also need the env variable to be able to differentiate between staging and production environments. When we render Codebytes from these places, it would just refer to the host environment variable and be able to differentiate between staging and production. 

I'm happy to go with 1, and avoid extra work. I'm just curious if this is ok as a temp solution. Curious what someone on web-plat might think @jakemhiller

Reply:
the only env var we should rely on being there is `NODE_ENV`, since it's a common standard for JS modules to use in modules. Adding new ENV vars requires each client to set up their build system to expose those vars, which isn't something we really want to do. 

That said, It's probably better than hard-coding it to production.

The ideal solution would be to initialize this in a way where the consuming application can pass these variables down through the component, or export a tracker creation function where you can configure it and pass down the tracker to the component.

```ts
export const createCodebytesTracker = (apiUrl: string) => createTracker(...);
```

Since you're importing `eventTracking` in other modules and using it outside of the context of the component that would probably require some refactoring... I think it's probably worth it though. Happy to do some pairing here to avoid taking up a lot of time, if that would help!